unoR3/unoR3.ino

```diff
@@ -44,27 +52,26 @@ boolean door_open = false;
 unsigned long heater_on_ms = 0;
 unsigned long last_temp_check = 0;
 unsigned long last_door_check = 0;
-unsigned long door_light_timer = 0;
 unsigned long mirror_light_on_ms = 0;
 
 String inputString = "";         // a string to hold incoming data
-String heater_ctrl_on = "HTRON\n";
-String heater_ctrl_off = "HTROFF\n";
-String main_ctrl_on = "MNLGTON\n";
-String main_ctrl_off = "MNLGTOFF\n";
-String right_lamp_ctrl_on = "RLMPON\n";
-String right_lamp_ctrl_off = "RLMPOFF\n";
-String left_lamp_ctrl_on = "LLMPON\n";
-String left_lamp_ctrl_off = "LLMPOFF\n";
+const String heater_ctrl_on = "HTRON\n";
+const String heater_ctrl_off = "HTROFF\n";
+const String main_ctrl_on = "MNLGTON\n";
+const String main_ctrl_off = "MNLGTOFF\n";
+const String right_lamp_ctrl_on = "RLMPON\n";
+const String right_lamp_ctrl_off = "RLMPOFF\n";
+const String left_lamp_ctrl_on = "LLMPON\n";
+const String left_lamp_ctrl_off = "LLMPOFF\n";
```

Comment:
made these `const` as they are never going to change. by doing this it means you can't accidentally change them elsewhere in the code